<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FileManagementTabForm</class>
 <widget class="QWidget" name="FileManagementTabForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1105</width>
    <height>527</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QToolButton" name="toolButtonPrevious">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="minimumSize">
          <size>
           <width>31</width>
           <height>31</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>31</width>
           <height>31</height>
          </size>
         </property>
         <property name="text">
          <string>...</string>
         </property>
         <property name="icon">
          <iconset theme="QIcon::ThemeIcon::GoPrevious"/>
         </property>
         <property name="iconSize">
          <size>
           <width>11</width>
           <height>11</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="toolButtonNext">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="minimumSize">
          <size>
           <width>31</width>
           <height>31</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>31</width>
           <height>31</height>
          </size>
         </property>
         <property name="text">
          <string>...</string>
         </property>
         <property name="icon">
          <iconset theme="QIcon::ThemeIcon::GoNext"/>
         </property>
         <property name="iconSize">
          <size>
           <width>11</width>
           <height>11</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="toolButtonUp">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="minimumSize">
          <size>
           <width>31</width>
           <height>31</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>31</width>
           <height>31</height>
          </size>
         </property>
         <property name="text">
          <string>...</string>
         </property>
         <property name="icon">
          <iconset theme="QIcon::ThemeIcon::GoUp"/>
         </property>
         <property name="iconSize">
          <size>
           <width>11</width>
           <height>11</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="toolButtonRefresh">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="minimumSize">
          <size>
           <width>31</width>
           <height>31</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>31</width>
           <height>31</height>
          </size>
         </property>
         <property name="text">
          <string>...</string>
         </property>
         <property name="icon">
          <iconset theme="QIcon::ThemeIcon::SystemReboot"/>
         </property>
         <property name="iconSize">
          <size>
           <width>11</width>
           <height>11</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="comboBoxPath">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>400</width>
           <height>30</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>30</height>
          </size>
         </property>
         <property name="editable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="toolButtonGo">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="minimumSize">
          <size>
           <width>31</width>
           <height>31</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>31</width>
           <height>31</height>
          </size>
         </property>
         <property name="text">
          <string>...</string>
         </property>
         <property name="icon">
          <iconset theme="QIcon::ThemeIcon::MailSend"/>
         </property>
         <property name="iconSize">
          <size>
           <width>11</width>
           <height>11</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonSwitch">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>71</width>
         <height>31</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>31</height>
        </size>
       </property>
       <property name="text">
        <string>切换起始目录</string>
       </property>
       <property name="icon">
        <iconset theme="QIcon::ThemeIcon::GoHome"/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="pushButtonDeselectAll">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>71</width>
         <height>31</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>31</height>
        </size>
       </property>
       <property name="text">
        <string>取消全选</string>
       </property>
       <property name="icon">
        <iconset theme="QIcon::ThemeIcon::ApplicationExit"/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonSelectAll">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>71</width>
         <height>31</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>31</height>
        </size>
       </property>
       <property name="text">
        <string>全选</string>
       </property>
       <property name="icon">
        <iconset theme="QIcon::ThemeIcon::EditSelectAll"/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonDelete">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>71</width>
         <height>31</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>31</height>
        </size>
       </property>
       <property name="text">
        <string>删除</string>
       </property>
       <property name="icon">
        <iconset theme="QIcon::ThemeIcon::EditDelete"/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonCopy">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>71</width>
         <height>31</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>31</height>
        </size>
       </property>
       <property name="text">
        <string>复制</string>
       </property>
       <property name="icon">
        <iconset theme="QIcon::ThemeIcon::EditCopy"/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonPaste">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>71</width>
         <height>31</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>31</height>
        </size>
       </property>
       <property name="text">
        <string>粘贴</string>
       </property>
       <property name="icon">
        <iconset theme="QIcon::ThemeIcon::EditPaste"/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonCut">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>71</width>
         <height>31</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>31</height>
        </size>
       </property>
       <property name="text">
        <string>剪切</string>
       </property>
       <property name="icon">
        <iconset theme="QIcon::ThemeIcon::EditCut"/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonUpload">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>71</width>
         <height>31</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>31</height>
        </size>
       </property>
       <property name="text">
        <string>上传</string>
       </property>
       <property name="icon">
        <iconset theme="QIcon::ThemeIcon::GoUp"/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonDownload">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>71</width>
         <height>31</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>31</height>
        </size>
       </property>
       <property name="text">
        <string>下载</string>
       </property>
       <property name="icon">
        <iconset theme="QIcon::ThemeIcon::GoDown"/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonOpen">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>71</width>
         <height>31</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>31</height>
        </size>
       </property>
       <property name="text">
        <string>打开</string>
       </property>
       <property name="icon">
        <iconset theme="QIcon::ThemeIcon::FolderOpen"/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonEdit">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>71</width>
         <height>31</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>31</height>
        </size>
       </property>
       <property name="text">
        <string>编辑</string>
       </property>
       <property name="icon">
        <iconset theme="QIcon::ThemeIcon::MailMessageNew"/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonCreateDir">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>71</width>
         <height>31</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>31</height>
        </size>
       </property>
       <property name="text">
        <string>创建文件夹</string>
       </property>
       <property name="icon">
        <iconset theme="QIcon::ThemeIcon::ListAdd"/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonRename">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>71</width>
         <height>31</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>31</height>
        </size>
       </property>
       <property name="text">
        <string>重命名</string>
       </property>
       <property name="icon">
        <iconset theme="QIcon::ThemeIcon::ApplicationExit"/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonProperty">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>71</width>
         <height>31</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>31</height>
        </size>
       </property>
       <property name="text">
        <string>属性</string>
       </property>
       <property name="icon">
        <iconset theme="QIcon::ThemeIcon::EditSelectAll"/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBoxSort">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>31</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>31</height>
        </size>
       </property>
       <property name="duplicatesEnabled">
        <bool>false</bool>
       </property>
       <item>
        <property name="text">
         <string>按名称升序</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>按名称降序</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>按时间升序</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>按时间降序</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>按类型升序</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>按类型降序</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>按大小升序</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>按大小降序</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Orientation::Vertical</enum>
     </property>
     <widget class="FileTreeView" name="treeView">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>300</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::Shape::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Shadow::Plain</enum>
      </property>
      <property name="selectionMode">
       <enum>QAbstractItemView::SelectionMode::ExtendedSelection</enum>
      </property>
      <property name="sortingEnabled">
       <bool>false</bool>
      </property>
      <property name="animated">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QListWidget" name="listWidget">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>100</height>
       </size>
      </property>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>FileTreeView</class>
   <extends>QTreeView</extends>
   <header location="global">FileTreeView.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
